src/signals/incident/containers/KtoContainer/components/KtoForm/index.js

```diff
@@ -215,23 +218,57 @@ const KtoForm = ({ options, isSatisfied, onSubmit }) => {
         />
       </GridArea>
 
-      <GridArea>
-        <StyledLabel id="subtitle-allows-contact">
-          Mogen wij contact met u opnemen naar aanleiding van uw feedback?{' '}
-          <Optional>(niet verplicht)</Optional>
-        </StyledLabel>
-
-        <CheckboxWrapper inline htmlFor="allows-contact">
-          <Checkbox
-            data-testid="ktoAllowsContact"
-            id="allows-contact"
-            aria-describedby="subtitle-allows-contact"
-            name="allows-contact"
-            onChange={onChangeAllowsContact}
-          />
-          Ja
-        </CheckboxWrapper>
-      </GridArea>
+      {satisfactionIndication === 'nee' && (
+        <>
+          <GridArea>
+            <StyledLabel htmlFor="text_extra">
+              {"Foto's toevoegen? "}
+              <Optional>(niet verplicht)</Optional>
+            </StyledLabel>
+            <HelpText id="subtitle-kto">
+              Voeg een foto toe om de situatie te verduidelijken.
+            </HelpText>
+            <FileInput
+              handler={() => {}}
+              parent={{ meta: { updateIncident: () => {} } }}
```

Comment:
empty, do we need to update the incident when in KTO? The incident seems to be final, so leave this empty for now.